index.html

```diff
@@ -32,7 +32,7 @@
     </div>
     <div id="log">
     </div>
-    <input id="command" placeholder="enter command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
+    <input id="command" placeholder="enter aircraft command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
```

Comment:
updated with develop reference the merge of #620 (refactor #619), so this file no longer exists here. :+1: